src/control/server/instance_storage.go

```diff
@@ -99,7 +99,7 @@ func (ei *EngineInstance) awaitStorageReady(ctx context.Context, skipMissingSupe
 	}
 	ei.log.Debugf("needsScmFormat: %t", needsScmFormat)
 
-	if !needsMetaFormat && ei.storage.ControlMetadataHasPath() && needsScmFormat {
+	if !needsMetaFormat && ei.storage.ControlMetadataPathConfigured() {
```

Comment:
Is there a permitted case where there is no SCM tier/no tmpfs? That case would fail getting the SCM config below.

Reply:
SCM tier is required

Reply:
Hm, okay. I was under the impression MD on SSD mode would eventually stop requiring this tier.